frames/src/main/java/com/checkout/frames/utils/extensions/PaymentFormStyleMapperExtensions.kt

```diff
@@ -34,48 +34,52 @@ internal fun String.provideText(text: String? = null): String {
     return if (text.isNullOrEmpty()) this else text
 }
 
-internal fun Int.provideTextId(textID: Int? = null): Int {
-    return textID ?: this
+internal fun Int.provideTextId(text: String? = null, textId: Int?): Int? {
+    /**
+     * Check titleText is provided from the merchant return null for the textID
+     * If titleText is not provided from the merchant return textID from the merchant or default style textId
+     */
+    return if (text.isNullOrBlank()) textId ?: this else null
```

Comment:
The current approach is totally opposite to what we have in styles.
It would be nice to use the same approach when `id` has priority in comparison with `text`.

If both `id` and `text` are provided then `id` will be used,
if only `text` is provided then `text` will be used,
if only `id` is provided then `id` will be used.

It can be 
`return if (!text.isNullOrEmpty()) textId else textId ?: this`

This gives us an old priority and at the same time, if neither text nor id was provided, the default id will be used.